.travis.yml

```diff
@@ -1,18 +1,15 @@
 ---
-sudo: false
-language: ruby
-cache: bundler
+sudo: required
+services: docker
+addons:
+  apt:
+    packages:
+      - docker-ce
 matrix:
   include:
-  - rvm: jruby-9.1.13.0
-    env: LOGSTASH_BRANCH=master
-  - rvm: jruby-9.1.13.0
-    env: LOGSTASH_BRANCH=6.x
-  - rvm: jruby-9.1.13.0
-    env: LOGSTASH_BRANCH=6.0
-  - rvm: jruby-1.7.27
-    env: LOGSTASH_BRANCH=5.6
+  - env: ELASTIC_STACK_VERSION=5.6.13
+  - env: ELASTIC_STACK_VERSION=6.4.3
+  - env: ELASTIC_STACK_VERSION=7.0.0-alpha1-SNAPSHOT
```

Comment:
would there be a way to specify only a minor version of the stack and have CI choose the latest? because otherwise this will require updating on every stack release which does not scale/not maintainable. WDYT?

Reply:
Updated docker testing to be the same as https://github.com/logstash-plugins/logstash-filter-memcached/pull/2